sample_kotlin/build.gradle

```diff
@@ -15,6 +15,10 @@ android {
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
     }
     buildTypes {
+        debug {
+            minifyEnabled false
```

Comment:
You don't need the minify bit in debug, maybe turn it on in release?

Reply:
I turned this on when testing locally: it was easier to test the proguard this way, without going through the hassle of doing a release build.